addons/event/report/event_event_templates.xml

```diff
@@ -155,6 +155,11 @@
     <div class="row page">
         <div t-attf-class="o_event_full_page_ticket_container page w-100 #{'o_event_full_page_ticket_responsive_html' if responsive_html else 'o_event_full_page_ticket'}">
             <div class="o_event_full_page_ticket_wrapper">
+                <div t-if="responsive_html" class="d-flex justify-content-center">
+                    <a class="btn o_event_download_ticket_btn mb-2" title="Download Tickets" t-attf-href="/event/{{ event.id }}/my_tickets?registration_ids={{ attendee.ids }}&amp;tickets_hash={{ event._get_tickets_access_hash(attendee.ids) }}">
```

Comment:
Isn't the URL used here literally the same as the one used to access this template, but with "responsive_html=0" ?

Would there be a (simple) way to reload the page and just remove that param from the URL then? Be it with a one-liner of JavaScript or by passing the tickets_download URL from the controller and manipulating the URL to remove this param with werkzeug URL tools?

That would prevent having to re-hash the event / tickets, not a huge deal, but it would be a bit more obvious.
(If it's too complex / takes too many lines of code, we can consider keeping this).

Reply:
Hello @awa-odoo 
I tried to do it the easy way but didn't find one :sweat_smile:

Reply:
Ok, at least we tried, let's go then 🚀